docs/ref/django-admin.txt

```diff
@@ -1497,6 +1497,15 @@ don't.
 Runs only tests :ref:`marked with the specified tags <topics-tagging-tests>`.
 May be specified multiple times and combined with :option:`test --exclude-tag`.
 
+.. note::
+
+    Tests that fail to load are always considered matching. This prevents tests
+    from falsely passing due, for example, to syntax errors.
```

Comment:
```suggestion
    Tests that fail to load are always considered matching. This prevents tests
    with syntax errors, for example, from falsely passing.
```

I'm not sure we need to call this out in a note, or add the `This prevents...`. I might just add `Tests that fail to load are always considered matching.` to the main text and leave it at that.

Reply:
Agreed, many thanks!